src/rosdiscover/Issue74.py

```diff
@@ -0,0 +1,102 @@
+import time
+import collections
+import sys
+
+sys.path.append("/interpreter")
+from summary import NodeSummary
+
+
+from loguru import logger
+
+import roswire
+
+f = open("Arch.yml", "w")
+
+# enable logging
+logger.enable('roswire')
+
+
+# first, we create a new ROSWire session
+rsw = roswire.ROSWire()
+
+def toString(l): 
+    s = ''
+    l1 = list(l)
+    if len(l1) == 0: 
+        return '[]'
+    for i in l1:
+        (fmt, topic) = i
+        s = s + "\n  - format: " + fmt + "\n    name: " + topic + "\n" 
+    return s
+
+# we then launch a container for a given robot.
+# once the end of the given context is reached, including the event where an
+# exception occurs, the container will be automatically destroyed.
+image = 'therobotcooperative/turtlebot3'
```

Comment:
Rewrite to take these in as arguments - perhaps take in a config yaml the same way that rosdiscover does, and get it that way.